src/site/content/en/blog/codelab-signin-form/index.md

```diff
@@ -30,89 +32,134 @@ As you'll see, these elements enable built-in browser functionality, improve acc
+
+Here's how your `index.html` should look at this point:
+
+<div class="glitch-embed-wrap" style="height: 480px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/login-form-best-practices-codelab-step-1?path=index.html&attributionHidden=true&sidebarCollapsed=true"
+    alt="How the Glitch should look after completing step 1."
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+Click **View App** to preview your login form. 
+The HTML you just added is valid and correct, but the default browser styling
+means it looks terrible and it's hard to use, especially on mobile.
+
+Click **View Source** to return to your source code.
 
 ## Step 2: Design for fingers and thumbs
 
-The HTML in Step 1 is valid and correct, but the default browser styling means it looks terrible and it's hard to use, especially on mobile.
+TODO(kayce): One-sentence description of the goal here. 
 
-Copy and paste the [complete CSS from step 2](https://glitch.com/edit/#!/signin-form-codelab-2?path=style.css) into [style.css](https://glitch.com/edit/#!/signin-form-codelab?path=style.css).
+Copy and paste the following CSS into your `style.css`:
+
+<div class="glitch-embed-wrap" style="height: 480px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/login-form-best-practices-codelab-step-2?path=style.css&attributionHidden=true&sidebarCollapsed=true"
+    alt="The CSS to copy-paste."
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+Click **View App** to check out your freshly styled login form, and then
+click **View Source** to return to `style.css`.
 
 That's quite a lot of code! The main things to be aware of are the changes to sizes:
-* Padding and margin are added to inputs.
-* Font-size is set for mobile and desktop.
 
-Styles are also set using the `:invalid` selector for when the email input has an invalid value.
+* `padding` and `margin` are added to inputs.
+* `font-size` is different for mobile and desktop.
+
+The `:invalid` selector is used to indicate when an input has an invalid value.
+
+The CSS layout is mobile-first:
 
-The CSS layout is mobile-first: 
-* The default CSS is for viewports less than 450 px wide.
-* The media query section sets overrides for viewports that are at least 450 px wide.
-* There are CSS variables for mobile and desktop font size.
+* The default CSS is for viewports less than `450px` wide.
+* The media query section sets overrides for viewports that are at least `450px` wide.
 
 It's very important at this point to test your code on real devices on desktop and mobile:
-* Is label and input text readable, especially for people whose vision isn't 20/20?
-* Are the inputs and **Sign in** button large enough to use as touch targets for thumbs?
 
-{% Aside 'codelab' %}
-[View complete code for Step 2](https://glitch.com/edit/#!/signin-form-codelab-2).
-{% endAside %}
+* Is label and input text readable, especially for people with low vision?
```

Comment:
"low vision" is the preferred phrase I believe

Reply:
Yeah — I actually wanted to capture the idea of *many* people, not just those who would say they have low vision. 

I have pretty good vision, for example, but I struggle to easily read text in many signin forms (especially on mobile or when visibility is poor).

Maybe just 'Is label and input text readable for all your users?'